src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -311,6 +347,9 @@ public long getIdLong()
     @Override
     public String getJumpUrl()
     {
+        if (isEphemeral())
+            throw new IllegalStateException("Cannot get the jump URL from ephemeral messages.");
```

Comment:
Why not?

Reply:
I thought ephemeral messages had no idea but they do and the link also works, will remove